x-pack/plugins/endpoint/common/types.ts

```diff
@@ -51,12 +55,22 @@ export interface AlertResultList {
   /**
    * The index of the requested page, starting at 0.
    */
-  request_page_index: number;
+  request_page_index: Maybe<number>;
```

Comment:
@oatkiller I thiiiiiink you're right. Still learning Typescript. I believe I intended `void` to include `undefined` and `null`, but since we do strict null checking, that's not the case?